R/server.R

```diff
@@ -1,5 +1,35 @@
 server <- function(elts, opt, custom_admin_panel) {
   function(input, output, session) {
+
+    # Get user information
+    user_info <- shiny::reactive({
```

Comment:
Maybe this reactive should only be executed if `get_user_info` is True?

Reply:
I did actually have that in and took it out when debugging. Good point!